### DISCUSSION TITLE
I can't understand a pinch of what your content says.

### DISCUSSION THREAD

**Discussion by KeniDouglas** [RakeshPotnuru/Blog#23] [Q&A]
Why not create a YouTube video explaining this in details, please 🙏.

**Comment by RakeshPotnuru** [comment]
Hey! Sorry for that. You need some technical knowledge to set this up. 
If you want a personal blog you can try ghost.org or Hashnode.

**Reply by KeniDouglas** [reply]
Can I link adsterra to any of this blog methods hosted?

**Reply by RakeshPotnuru** [reply]
Yes I think. You can insert code.
